pkg/controller/cheinstallation/cheinstallation_controller.go

```diff
@@ -149,19 +143,18 @@ func (r *ReconcileCheInstallation) ensureCheNamespace(logger logr.Logger, cheIns
 		}
 		return false, err
 	}
-
-	// To handle if namespace is deleted by user and it's in terminating state and not in active state
 	if ns.Status.Phase != v1.NamespaceActive {
-		return false, errs.Errorf("namespace %s is not in active state", ns.Name)
+		logger.Info("namespace is not in active state", "namespace", ns.Name, "phase", ns.Status.Phase)
+		return true, nil
 	}
 	return false, nil
 }
 
-func (r *ReconcileCheInstallation) ensureCheOperatorGroup(logger logr.Logger, ns string, cheInstallation *v1alpha1.CheInstallation) (bool, error) {
+func (r *ReconcileCheInstallation) ensureCheOperatorGroup(logger logr.Logger, cheInstallation *v1alpha1.CheInstallation) (bool, error) {
```

Comment:
If possible, don't do this refacotoring to drop `ns` param (in both methods).  Due to this, there are lot of changes in test_code also and later I need to resolve merge conflict for other PR on CheInstallation.

Reply:
what merge conflicts are you talking about? This `ns` parameter is not needed in these 2 funcs, so I believe we should remove it.

Reply:
As I said, if possible then don't do this refactoring.  I have not thought ahead whether this will bring any merge conflict or not.  In general, I am asking to avoid (if possible) refactoring on CheInstallation code as multiple team members are working on same code base but fine if you want to do this refactoring.